numpy/ma/tests/test_core.py

```diff
@@ -2998,6 +2998,10 @@ def test_take(self):
         assert_equal(x.take([[0, 1], [0, 1]]),
                      masked_array([[10, 20], [10, 20]], [[0, 1], [0, 1]]))
 
+        # assert_equal crashes when passed np.ma.mask
```

Comment:
This raises the question of whether any of these tests are actually verifying things, since `assert_equal` will ignore the mask